test/e2e/dra/dra.go

```diff
@@ -1911,6 +1921,175 @@ var _ = framework.SIGDescribe("node")(framework.WithLabel("DRA"), framework.With
+			pod.Spec.Containers[1].Name = "container2"
+
+			res := v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName)] = resource.MustParse("1")
+			pod.Spec.Containers[0].Resources.Requests = res
+			pod.Spec.Containers[0].Resources.Limits = res
+			res = v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName2)] = resource.MustParse("1")
+			pod.Spec.Containers[1].Resources.Requests = res
+			pod.Spec.Containers[1].Resources.Limits = res
+			res = v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName3)] = resource.MustParse("1")
+			pod.Spec.Containers[2].Resources.Requests = res
+			pod.Spec.Containers[2].Resources.Limits = res
+
+			b.create(ctx, pod)
+			err := e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, pod)
+			framework.ExpectNoError(err, "start pod")
+			containerEnv := []string{
+				"container_0_request_0", "true",
+				"container_1_request_0", "true",
+				"container_2_request_0", "true",
+			}
+			testContainerEnv(ctx, f, pod, pod.Spec.Containers[0].Name, false, containerEnv...)
+		})
+		ginkgo.It("must run a pod with extended resource", ginkgo.Label("dra extended resource single-node three-containers"), func(ctx context.Context) {
+			pod := b.pod()
+			pod.Spec.Containers = append(pod.Spec.Containers, *pod.Spec.Containers[0].DeepCopy())
+			pod.Spec.Containers = append(pod.Spec.Containers, *pod.Spec.Containers[0].DeepCopy())
+			pod.Spec.Containers[0].Name = "container0"
+			pod.Spec.Containers[1].Name = "container1"
+			pod.Spec.Containers[1].Name = "container2"
+
+			res := v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName)] = resource.MustParse("1")
+			pod.Spec.Containers[0].Resources.Requests = res
+			pod.Spec.Containers[0].Resources.Limits = res
+			res = v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName2)] = resource.MustParse("1")
+			res[v1.ResourceName(extendedResourceName3)] = resource.MustParse("1")
+			pod.Spec.Containers[1].Resources.Requests = res
+			pod.Spec.Containers[1].Resources.Limits = res
+			res = v1.ResourceList{}
+			res[v1.ResourceName(extendedResourceName4)] = resource.MustParse("1")
+			res[v1.ResourceName(extendedResourceName5)] = resource.MustParse("1")
+			res[v1.ResourceName(extendedResourceName6)] = resource.MustParse("1")
+			pod.Spec.Containers[2].Resources.Requests = res
+			pod.Spec.Containers[2].Resources.Limits = res
+
+			b.create(ctx, pod)
+			err := e2epod.WaitForPodRunningInNamespace(ctx, f.ClientSet, pod)
+			framework.ExpectNoError(err, "start pod")
+			containerEnv := []string{
+				"container_0_request_0", "true",
+				"container_1_request_0", "true",
+				"container_1_request_1", "true",
+				"container_2_request_0", "true",
+				"container_2_request_1", "true",
+				"container_2_request_2", "true",
+			}
+			testContainerEnv(ctx, f, pod, pod.Spec.Containers[0].Name, false, containerEnv...)
+		})
+	})
+
+	framework.Context(f.WithFeatureGate(features.DRAExtendedResource), "multiple node", func() {
```

Comment:
Consider the full test name. 
```suggestion
	framework.Context(f.WithFeatureGate(features.DRAExtendedResource), "multiple nodes", func() {
```

=> "[It] [sig-node] [DRA] [FeatureGate:DynamicResourceAllocation] [Beta] [Feature:OffByDefault] [FeatureGate:DRAExtendedResource] [Alpha] [Feature:OffByDefault] multiple nodes must run a pod with extended resource [Serial]"